salt/modules/heat.py

```diff
@@ -496,11 +496,19 @@ def create_stack(name=None, template_file=None, enviroment=None,
 
         salt '*' heat.create_stack name=mystack \\
                  template_file=salt://template.yaml \\
-                 enviroment=salt://enviroment.yaml \\
+                 environment=salt://environment.yaml \\
                  parameters="{"image": "Debian 8", "flavor": "m1.small"}" \\
                  poll=5 rollback=False timeout=60 profile=openstack1
 
+    .. versionadded:: 2017.7.5,2018.3.1
+
+        The spelling mistake in parameter `enviroment` was corrected to `environment`.
+        The misspelled version is still supported for backward compatibility.
+
     '''
+    if environment is None:
+        # Support for misspelled parameter
+        environment = enviroment
```

Comment:
I think we should put the misspelled version on a deprecation path. Look for the `warn_util` function of how to do this and we can remove it in 2 feature releases.